packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -30,6 +30,14 @@ class CreateCommand extends Command {
       defaultsTo: true,
       help: 'Whether to run "pub get" after the project has been created.'
     );
+    argParser.addFlag(
+      'with-driver-test',
```

Comment:
Do you have a sense for whether projects will have largely unit tests, or largely driver tests, or some mix of both?

I'm just wondering when (or whether) we should default this to true, and / or remove the flag completely and just generate a test for all apps.